### PR TITLE
docs: add release announcement post

### PR DESCRIPTION
**Pull request by desimone**
- add mailchimp newsletter form

https://deploy-preview-617--pomerium-docs.netlify.app/
https://deploy-preview-617--pomerium-docs.netlify.app/posts/2020/04/16/release-0-7/

Signed-off-by: Bobby DeSimone <bobbydesimone@gmail.com>

## Summary
Adds v0.7 release docs / "announcement" like functionality to docs. Also adds simple mailchimp newsletter signup. 

**Checklist**:
- [x] updated docs
- [x] ready for review

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -5,14 +5,33 @@ module.exports = {
...
+        {
+          id: "post",
+          dirname: "_posts",
+          path: "/posts/",
+          itemPermalink: "/posts/:year/:month/:day/:slug",
+        },
+      ],
+    },
+    "vuepress-plugin-mailchimp": {
+      title: "Sign up to be notified of new features and updates",
```
Comment (travisgroth) [inline docs/.vuepress/config.js:24]: 👍

File: docs/_posts/2020-04-16-release-0-7.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+author: "Bobby DeSimone"
+---
+
+# Announcing Pomerium 0.7
+
+The 0.7 release of Pomerium lays the groundwork to support rich, dynamic access policies capable of making authorization decisions based on external data-sources from outside your identity provider.
+
+This release includes the following features:
+
+- **Open Policy Agent**: Pomerium now leverages Cloud Native Computing Foundation's [Open Policy Agent](https://www.openpolicyagent.org/) (OPA) as the default policy engine. OPA support in this version clears the pathway to enabling operators to write authorization policy in a high-level, declarative language that promotes safe, performant, fine-grained controls that incorporate data from external records of account. Access control policies will continue to work as before, and are now backed by an even faster evaluation engine.
```
Comment (travisgroth) [inline docs/_posts/2020-04-16-release-0-7.md:17]: >clears the pathway to enabling operators

Might read as a current feature.  Also a bit wordy.  Maybe something like

>OPA support in this version unlocks future capabilities for writing authorization policy in the Rego policy language.  This will promote safe, performant, fine-grained controls to incorporate data from external records of account.

Comment (desimone) [reply docs/_posts/2020-04-16-release-0-7.md:17]: Agree, and well put.
